docs/kickstart-docs.rst

```diff
@@ -593,6 +593,15 @@ graphical
 Perform the kickstart installation in graphical mode. This is the
```

Comment:
Cannot add a comment for the commit message so I'm adding it here. First of all, it's an option not a param(eter). And we don't have graphical modes, we only have one graphical mode and a text mode. Sorry for nitpicking here, but I really find the commit message confusing.